Gemfile

```diff
@@ -36,13 +36,16 @@ group :development, :test do
   gem 'brakeman', '~> 4.1', require: false
   gem 'byebug', '~> 9.1'
   gem 'dotenv-rails', '~> 2.2', '>= 2.2.1'
+  gem 'factory_bot_rails', '~> 4.8', '>= 4.8.2'
```

Comment:
Moving this gem to the `:development, :test` groups according to the FactoryBot documentation.